### PR TITLE
Release step and reverting to version 0.1.0

### PR DESCRIPTION
**Pull request by suztomo**
Formatted release procedure is available in https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/revert_0.1.0/RELEASING.md

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+This script creates a branch and a tag with the version number.
+
+### Reverting prepare_release.sh
+
+Sometimes subsequent builds fail and you want to start over with the same version number with
+updated master by reverting this step.
+To revert the effect of this step, delete the tag and the branch. For example, to revert version
+'0.1.1', do the following steps:
```
Comment (elharo) [inline RELEASING.md]: You shouldn't need to delete the tag. Just create v1.1.2A, v1.1.2B, etc.

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+|dependencies-parent| cloud-java-tools-cloud-opensource-java-parent-kokoro-release|
+|linkage-checker-enforcer-rules|cloud-java-tools-cloud-opensource-java-enforcer-rules-kokoro-release|
+
+The order is not strict and they can run concurrently.
+
+When asked in Rapid, specify the tag (vX.Y.Z) as 'committish'.
+
+## 4. Confirm the builds
+
+The artifacts should be available in Sonatype staging repository.
```
Comment (elharo) [inline RELEASING.md]: How do you confirm this?

File: RELEASING.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## 3. Initiate Rapid builds
+
+| ArtifactId | Rapid project name |
+| ---------- | ------------------ |
+|dependencies| cloud-java-tools-cloud-opensource-java-dependencies-kokoro-release|
+|dependencies-parent| cloud-java-tools-cloud-opensource-java-parent-kokoro-release|
+|linkage-checker-enforcer-rules|cloud-java-tools-cloud-opensource-java-enforcer-rules-kokoro-release|
+
+The order is not strict and they can run concurrently.
```
Comment (elharo) [inline RELEASING.md]: I'm not sure this is true. Dependents should appear before dependencies; i.e.

1. Parent
2. dependencies
3. enforcer-rule

File: RELEASING.md (hunk 1 above)
Comment (suztomo) [reply RELEASING.md]: That makes difficult to find corresponding commit in [releases page](https://github.com/GoogleCloudPlatform/cloud-opensource-java/releases) from Maven central version.

File: RELEASING.md (hunk 3 above)
Comment (suztomo) [reply RELEASING.md]: Updated the order in the table. But the builds do not need to wait for dependents. Clarified the sentence.

File: RELEASING.md (hunk 2 above)
Comment (suztomo) [reply RELEASING.md]: This part is removed in favor of the ones written in ct4j doc.

File: RELEASING.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Release Steps
+
+To release artifacts 'dependencies-parent', 'dependencies', and 'linkage-checker-enforcer-rules',
+follow the "Developers/Releasing" steps in go/ct4j.
```
Comment (elharo) [inline RELEASING.md]: Please link directly to https://g3doc.corp.google.com/company/teams/cloud-java/tools/developers/releasing.md?cl=head and update that page as needed.

File: RELEASING.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## Rapid builds
+
+| ArtifactId | Rapid project name | Dependency |
+| ---------- | ------------------ | --------- |
+|dependencies-parent| cloud-java-tools-cloud-opensource-java-parent-kokoro-release||
+|dependencies| cloud-java-tools-cloud-opensource-java-dependencies-kokoro-release|dependencies-parent|
+|linkage-checker-enforcer-rules|cloud-java-tools-cloud-opensource-java-enforcer-rules-kokoro-release|dependencies-parent, dependencies|
+
+To satisfy their dependency tree, their dependency need to be released together.
```
Comment (elharo) [inline RELEASING.md]: I don't understand what this sentences is trying to say.

File: RELEASING.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+The release pipelines can run concurrently. For example, you don't have to wait for
+`dependencies-parent` pipeline before initiating `dependencies` pipeline.
+
+## Merge PR to increment version at the last step
```
Comment (elharo) [inline RELEASING.md]: Is there some reason this is different than the other projects? Do they not need to do this?

File: RELEASING.md (hunk 4 above)
Comment (suztomo) [reply RELEASING.md]: Added the link.

File: RELEASING.md (hunk 6 above)
Comment (suztomo) [reply RELEASING.md]: Created cl/239195115 and asking @loosebazooka .
Until when the CL is submitted, I keep this part in this RELEASING.md.

File: RELEASING.md (hunk 5 above)
Comment (suztomo) [reply RELEASING.md]: Changed this to concrete sentences.

File: RELEASING.md (hunk 6 above)
Comment (suztomo) [reply RELEASING.md]: This section is removed. As discussed, we'll update the team's document.

**Pull request merged**